README.md

```diff
@@ -43,7 +43,7 @@ webpack-validator makes it easy:
 
 ![validation-example](https://cloud.githubusercontent.com/assets/3755413/14134087/b3279738-f654-11e5-9752-367b01ac123d.png)
 
-### Usage
+### Usage API
```

Comment:
I think calling the programmatic usage "API" is a bit confusing. Its just programmatic vs cli. I'd call this chapter "Usage" and then add a paragraph below the programmatic example where it says "You can also use webpack-validator as a command line utility by installing it with `npm i -g webpack-validator` and then running `webpack-validator <your-config-file>` on the command line.